src/ocrd/processor/builtin/dummy_processor.py

```diff
@@ -25,13 +27,16 @@ class DummyProcessor(Processor):
     Bare-bones processor creates PAGE-XML and optionally copies file from input group to output group
     """
 
-    def process_page_pcgts(self, *input_pcgts: OcrdPage, output_file_id: Optional[str] = None, page_id: Optional[str] = None) -> OcrdPageResult:
+    def process_page_pcgts(self, *input_pcgts: Optional[OcrdPage], page_id: Optional[str] = None) -> OcrdPageResult:
```

Comment:
same here (`Optional` does not seem right)

Reply:
> See [#8 (comment)](https://github.com/bertsky/core/pull/8#discussion_r1718551244)